tools/SetupFlow/DevHome.SetupFlow/ViewModels/RepoConfigViewModel.cs

```diff
@@ -17,6 +17,7 @@
 using DevHome.SetupFlow.Utilities;
 using Microsoft.Extensions.Hosting;
 using Microsoft.UI.Xaml;
+using WinUIEx;

```

Comment:
I'm guessing this got added automatically and overlooked.

Reply:
Correct.  Thank you for catching this.